html_css/v2/main-course/responsive-css/responsive-css-introduction.md

```diff
@@ -0,0 +1,26 @@
+### Introduction
+"Responsive Design" is an incredibly important concept when it comes to developing for the web. In many ways you are already equipped to create beautiful responsive websites. In this section we're going to fill in the gaps.
+
+### Learning Outcomes
+* You'll learn what 'Responsive Design' is and why it's important.
+
+### Terminology
+The phrase 'Responsive Design' is the most common phrase across the internet to refer to the concept we're talking about here. Unfortunately, it is a little bit misleading, because of the word 'design'. These lessons are not going to focus much on the actual _design_ of responsive websites, but the techniques you can use to implement responsivity on websites.
```

Comment:
Small term change only because I've not come across "responsivity" in regards to responsive design, but if they're interchangeable then disregard

```suggestion
The phrase 'Responsive Design' is the most common phrase across the internet to refer to the concept we're talking about here. Unfortunately, it is a little bit misleading, because of the word 'design'. These lessons are not going to focus much on the actual _design_ of responsive websites, but the techniques you can use to implement responsiveness on websites.
```